web/js/map/ui.js

```diff
@@ -337,15 +337,15 @@ export function mapui(models, config) {
   };
 
   /*
-   * Reloads layers on a change in layer order
+   * Reloads layer on change
    *
-   * @method updateLayerOrder
+   * @method updateLayer
    * @static
    *
    *
    * @returns {void}
    */
-  var updateLayerOrder = function () {
+  var updateLayer = function () {
```

Comment:
Why did you rename this function? `updateLayerOrder` and `updateLayer` mean different things. Is this function doing something different than it was before?

Reply:
If this function is just calling `reloadLayers`, why not use that directly, instead of having it wrapped inside another function?

Reply:
I tried using it directly, but it wasn't working.

Reply:
Also, see my comment above about the reason behind renaming this function.